main.yml

```diff
@@ -331,6 +332,17 @@ subscriptions:
       - /staging/subscriptions
   git_repo: https://github.com/RedHatInsights/curiosity-frontend
 
+topological-inventory:
+  title: Topological inventory
+  api:
+    versions:
+      - v1.0
```

Comment:
All other APIs are just `v1`. Is this one really `v1.0`?

Reply:
Yup, it is `v1.0` the API docs allow both.